.github/workflows/ci.yaml

```diff
@@ -52,6 +52,52 @@ jobs:
       - name: Run chart-testing (lint)
         run: ct lint --validate-maintainers=false --charts charts/vaas
 
+      - name: Install Minikube
+        uses: manusa/actions-setup-minikube@v2.13.0
+        with:
+          minikube version: v1.34.0
+          kubernetes version: v1.30.0
+          start args: --addons ingress
+
+      - name: Set up kubectl
+        uses: azure/setup-kubectl@v3
+        with:
+          version: 'latest'
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.13.3
+
+      - name: Deploy Helm Chart
+        run: |
+          cat <<EOF > values-local.yaml
+          global:
+            imagePullSecrets:
+              - registry
```

Comment:
IIRC this is unnecessary now, setting  just the dockerconfigjson should be fine.